rdmo/accounts/views.py

```diff
@@ -89,3 +91,18 @@ def token(request):
     return render(request, 'account/account_token.html', {
         'token': token
     })
+
+
+def shibboleth_login(request):
+    if request.user.is_authenticated:
+        return HttpResponseRedirect(reverse('projects'))
+    else:
+        login_url = settings.SHIBBOLETH_LOGIN_URL + f'?target={request.path}'
+        return HttpResponseRedirect(login_url)
+
+
+def shibboleth_logout(request):
+    logout_url = reverse('account_logout')
+    if request.user.username.endswith('@hu-berlin.de'):
```

Comment:
do you want to have this special case hardcoded? ;)

Reply:
urgs, I forgot...

Reply:
Fixed.